packages/module/src/AttachMenu/AttachMenu.scss

```diff
@@ -40,28 +41,27 @@
     overflow: hidden;
   }
 
+  .pf-v6-c-menu__item {
+    padding-inline-start: var(--pf-t--global--spacer--md);
+    padding-inline-end: var(--pf-t--global--spacer--md);
+  }
+
   .pf-v6-c-menu__item-icon {
-    --pf-v6-c-menu__item--icon--Color: #707070;
+    --pf-v6-c-menu__item--icon--Color: var(--pf-t--global--icon--color--subtle);
```

Comment:
Hi! I am wondering if we need to have the icon color for the menu items to be subtle(grey fill)? I know marks original had a custom grey color and the Pf token matches but we would use --pf-t--global--icon--color--regular ? wdyt?

Reply:
Not at all! It looks like --pf-t--global--icon--color--regular is the default to me, so I can just drop this and it should work.